tests/functional/src/main/java/org/mule/functional/junit4/runners/ClassLoaderRunnerFactory.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.functional.junit4.runners;
+
+/**
+ * Creates a {@link ClassLoader} to be used for running the JUnit test.
+ *
+ * @since 4.0
+ */
+public interface ClassLoaderRunnerFactory
```

Comment:
This interface seems redundant: is only implemented once, the implementation is created without a factory and there is no mock using it